lib/hooksData.js

```diff
@@ -0,0 +1,49 @@
+'use strict';
+
+const rp = require('request-promise-native')
+
+class HooksData {
+
+    constructor(taskId, user, pass) {
+        this.taskId = taskId;
+        this.user = user;
+        this.pass = pass;
+        this.basePath = process.env.ELASTICIO_API_URI || 'https://api.elastic.io/';
```

Comment:
The last slash must be removed due to the way the url is constructed down there.